docs/source/using/addons.adoc

```diff
@@ -194,6 +195,21 @@ the _eap_ add-on.
 NOTE: If two add-ons have the same priority the order in which they are getting applied is not
 determined.
 
+[[apply-addons]]
+== Applying add-ons
+
+Add-ons can be explicitly executed with the link:../command-ref/minishift_addons_apply{outfilesuffix}[`minishift addons apply`] command.
+You can use _add-on_ _apply_ command for both enabled and disabled add-ons. For applying multiple add-ons with a single command specify add-on names separated by space.
```

Comment:
I don't get the formatting here - _add-on_ _apply_. Actual commands we put into backticks. So do you mean `addons apply`? Or do you want to say "You can use add-on `apply` command", in which case you generically refer to the add-on mechanism. No need for emphasis here. Or maybe just "You can use the `apply` command"?